README.md

```diff
@@ -265,7 +278,7 @@ Find out more about [Setting up Sign in with Apple](https://auth0.com/docs/conne
 ### Authentication API (iOS / macOS / tvOS)
 
 The Authentication API exposes AuthN/AuthZ functionality of Auth0, as well as the supported identity protocols like OpenID Connect, OAuth 2.0, and SAML.
-We recommend using our Hosted Login Page but if you wish to build your own UI you can use our API endpoints to do so. However some Auth flows (Grant types) are disabled by default so you will need to enable them via your Auth0 Dashboard as explained in [this guide](https://auth0.com/docs/clients/client-grant-types#edit-available-grant_types).
+We recommend using Universal Login but if you wish to build your own UI you can use our API endpoints to do so. However some Auth flows (Grant types) are disabled by default so you will need to enable them via your Auth0 Dashboard as explained in [this guide](https://auth0.com/docs/clients/client-grant-types#edit-available-grant_types).
```

Comment:
> We recommend using Universal Login

Again let's provide a link here to some docs around Universal Login (probably this one: https://auth0.com/docs/universal-login)

Reply:
Done in aacf3ce99941bb8d7e85e0fd3361b367b77e0e77.